configs/vlasov-1d/bump-on-tail.yaml

```diff
@@ -67,6 +67,7 @@ terms:
   field: poisson
   edfdv: exponential
   time: sixth
+  diags: false
```

Comment:
I think it's better to use a list here which may be empty, see my other comments for how that can be useful.